scripts/install_third_party.py

```diff
@@ -324,7 +324,7 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
-MATHJAX_REV = '2.6.0'
+MATHJAX_REV = '2.6.7'
```

Comment:
Why not 2.7.5?

Also, maybe all this stuff can be deleted. The "too big" issue arises when we deploy stuff to App Engine, because there is a file limit. But I think this can be handled using skip_files in app.yaml, so maybe let's just do that there and get rid of this?

Reply:
(Deleting all this stuff, if possible, is better too because then you won't have accidental skew between this file and manifest.json.)

Reply:
Hi @seanlip, yes we can do that.
I'll have a brief look at it.

Also, @seanlip & @apb7, seem like MathsExpression version  2.0.0.alpha8 is not working as expected I've made a doc [here](https://docs.google.com/document/d/11nURWi1Y5pR8s2K3roW4BSYaKrQljGp1Tt55y6byVzE/edit). PTAL if the tests are correct and please help me confirm it. Thanks!

Reply:
I'm going to defer to @apb7 on this, but a general piece of advice: try not to upgrade to an "alpha" version; that is usually not stable. I suggest filing an issue on the math-expressions repo as well. Thanks!

Reply:
@seanlip, in app.yaml we skip the unnecessary files to deploy in app-engine but in the above code, we delete few folders from the MathJax folder like unpacked & fonts (size 20mb)
because the size of the MathJax folder is too big (Near to 67mb).
if we remove the above code and add the same to skip files we will still have those 20mb files

Reply:
But that's only on the local machine right? I think developers can self-manage that. I think the main reason it was added was before we knew that the skip_files option existed.

Reply:
we may also get issues with linter, as there is a third_party_size_check case which I believe is not aware of skip files in app.yaml. I've attached the file count difference below.

<img width="680" alt="Screenshot 2019-05-23 at 1 04 23 PM" src="https://user-images.githubusercontent.com/41017303/58234074-54d12780-7d5b-11e9-9d66-2760b660666f.png">

Reply:
Ah I see, yes, that could be an issue. OK, we may need to leave it then. But if there's a generic way to download a subset of the MathJax stuff that would be nice. Or alternatively if the linter can respect skip_files in app.yam, that works well too.

Btw do you really need sudo in front of the command? It should work without it... if not, that's a bug that should probably be filed.

Reply:
yes, we should find a way for this @apb7 may have suggestions.

> Btw do you really need sudo in front of the command? It should work without it... if not, that's a bug that should probably be filed.
>
No, sudo is not required. I'm currently using the commands from an unprivileged user so I need to use it.

Reply:
> Or alternatively if the linter can respect skip_files in app.yam, that works well too.

I think this can be done. @NishealJ, try looking into the implementation of this particular check. It won't be that difficult. Feel free to ping me if you face any problems. Thanks!

Reply:
Hi @apb7  & @seanlip,
As discussed, we have updated our linters to track the skip_files in app.yaml.
Hence, now we can remove the scripts which delete the unwanted MathJax dirs and add them in skip_files. I've done the needed  PTAL Thanks!

Below i've added the linter results of the file count before/after adding MathJax unwanted dirs into skip_files.

<img width="567" alt="Screenshot 2019-05-26 at 9 16 45 PM" src="https://user-images.githubusercontent.com/41017303/58403308-f9b37380-807f-11e9-9a05-2e5adf78cc84.png">